confgenerator/logging.go

```diff
@@ -20,6 +20,8 @@ import (
 	"github.com/GoogleCloudPlatform/ops-agent/confgenerator/fluentbit"
 )
 
+const HttpRequestKey = "logging.googleapis.com/httpRequest"
```

Comment:
I think this may be a good question to bring up on the bug itself, unfortunately I'm not sure. It does look like using this outside of the third party integrations does rely on a feature that is also undocumented, so I would be surprised if any customers were, but that's speculation.